healthworkforce/features/utils.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+const axios = require('axios')
+
+const OPENHIM_PROTOCOL = process.env.OPENHIM_PROTOCOL || 'http'
+const OPENHIM_API_HOSTNAME = process.env.OPENHIM_API_HOSTNAME || 'localhost'
+const OPENHIM_TRANSACTION_API_PORT =
+  process.env.OPENHIM_TRANSACTION_API_PORT || '5001'
+const CUSTOM_TOKEN_ID = process.env.CUSTOM_TOKEN_ID || 'test'
+
+const practitionerId = 'P10004'
+const practitionerRoleId = 'PR10001'
+const locationId = '2'
+const organizationId = '3'
+
+exports.triggerSync = async () => {
+  await axios({
+    url: `${OPENHIM_PROTOCOL}://${OPENHIM_API_HOSTNAME}:${OPENHIM_TRANSACTION_API_PORT}/mcsd-trigger`,
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+
+  // Allow the syncing to finish. The data will only be available in FHIR after a few seconds
+  await new Promise ((resolve, _reject) => {
+    setTimeout(() => resolve(), 60000)
```

Comment:
We could probably change this to be something smaller. Maybe optimise it once all is completed.

Reply:
addressed